### PR TITLE
 fix(Channel#startTyping): less breaking

### PR DESCRIPTION
**Pull request by Lewdcario**
**Please describe the changes this PR makes and why it should be merged:**
The method keeps the changes introduced in Master branch and stops after one rejection and doesn't return a Promise. Also rejects silently since it can't be caught by the end developer.

**Semantic versioning classification:**  
- [ ] This PR changes the library's interface (methods or parameters added)
  - [ ] This PR includes breaking changes (methods removed or renamed, parameters moved or removed)
- [ ] This PR **only** includes non-code changes, like changes to documentation, README, etc.

### CODE REVIEW

**Comment by KingDGrizzle** [comment]
Wasn't the point of the promise to reject when the user lost access to send? So you can catch it and stuff?

**Comment by Lewdcario** [comment]
The developer can't catch it himself so it should fail silently.

**Comment by appellation** [comment]
building on what @KingDGrizzle mentioned, it does seem pointless to create and store a Promise in this case.  simply catching the API call and all the associated cleanup logic should be sufficient.

File: src/structures/interfaces/TextBasedChannel.js [hunk 1]
```diff
@@ -271,23 +271,20 @@ class TextBasedChannel {
...
-          });
-        }, 9000),
-        resolve,
-      });
-      this.client.rest.methods.sendTyping(this.id).catch(() => {
-        this.client.clearInterval(entry.interval);
-        this.client.user._typing.delete(this.id);
-      });
-      this.client.user._typing.set(this.id, entry);
+    Object.assign(entry, {
```
Comment (SpaceEEC) [inline src/structures/interfaces/TextBasedChannel.js]: No need to `Object.assign` anymore.
It was only there to be able to access the entry in the now gone promise body.

**Pull request merged**